src/coreclr/jit/utils.cpp

```diff
@@ -3332,6 +3332,22 @@ uint32_t BitOperations::PopCount(uint64_t value)
 #endif
 }
 
+#ifdef HOST_UNIX
+//------------------------------------------------------------------------
+// BitOperations::PopCount: Returns the population count (number of bits set) of a mask.
+//
+// Arguments:
+//    value - the value
+//
+// Return Value:
+//    The population count (number of bits set) of value
+//
+uint32_t BitOperations::PopCount(unsigned __int128 value)
+{
+    return BitOperations::PopCount(static_cast<uint64_t>(value));
```

Comment:
it should. Currently, since we just use 64-bits, I want to quickly prototype and see the TP impact of changing the `regMaskTP` data type to 128-bits. I will be cleaning few things here and other places (e.g. `BitScanForward`) before marking it "ready for review".